v3/src/components/web-view/web-view.tsx

```diff
@@ -26,6 +30,68 @@ export const WebViewComponent = observer(function WebViewComponent({ tile }: ITi
       <div className="codap-web-view-iframe-wrapper">
         <iframe className="codap-web-view-iframe" ref={iframeRef} src={webViewModel.url} />
       </div>
+      <WebViewDropOverlay />
     </div>
   )
 })
+
+// The WebViewDropOverlay broadcasts notifications to plugins as the user drags and drops attributes.
+function WebViewDropOverlay() {
+  const overlayRef = useRef<HTMLElement|null>()
+  const { tile, tileId } = useTileModelContext()
+  const dropId = `web-view-drop-overlay-${tileId}`
+  const { active, setNodeRef } = useDroppable({ id: dropId })
+  const info = active && getDragAttributeInfo(active)
+  const dataSet = info?.dataSet
+  const attributeId = info?.attributeId
+  // Mouse x and y are tracked so we know where the mouse is when a dragged attribute is dropped.
+  const mouseX = useRef<number|undefined>()
+  const mouseY = useRef<number|undefined>()
+
+  useDropHandler(dropId, (_active: Active) => {
+    const { dataSet: dropDataSet, attributeId: dropAttributeId } = getDragAttributeInfo(_active) || {}
+    if (dropDataSet && dropAttributeId && mouseX.current != null && mouseY.current != null) {
+      tile?.applyModelChange(() => {}, {
+        notify: dragWithPositionNotification("drop", dropDataSet, dropAttributeId, mouseX.current, mouseY.current),
+        webViewId: tileId
+      })
+    }
+  })
+
+  if (!dataSet || !attributeId) return null
+
+  const handleMouseMove: MouseEventHandler<HTMLDivElement> = event => {
+    const { top, left } = overlayRef.current?.getBoundingClientRect() ?? { top: 0, left: 0 }
+    const x = event.clientX - left
+    const y = event.clientY - top
+
+    if (mouseX.current !== x || mouseY.current !== y) {
+      tile?.applyModelChange(() => {}, {
+        notify: dragWithPositionNotification("drag", dataSet, attributeId, x, y),
+        webViewId: tileId
+      })
+      mouseX.current = x
+      mouseY.current = y
+    }
+  }
+
+  const handleMouseEnterLeave = (operation: string) => {
+    tile?.applyModelChange(() => {}, {
+      notify: dragNotification(operation, dataSet, attributeId),
+      webViewId: tileId
+    })
+  }
+
+  const setRef = (ref: HTMLDivElement) => {
+    overlayRef.current = ref
+    setNodeRef(ref)
+  }
+
+  return <div
+    className="codap-web-view-drop-overlay"
+    onMouseEnter={() => handleMouseEnterLeave("dragenter")}
+    onMouseLeave={() => handleMouseEnterLeave("dragleave")}
+    onMouseMove={handleMouseMove}
```

Comment:
Did you try with pointer events instead of mouse events? With pointer events we should get touch support automatically. With mouse events, touch support might require some additional work.